vite.config.ts

```diff
@@ -0,0 +1,46 @@
+import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vite';
+import eslint from 'vite-plugin-eslint';
+
+const largeLibraries = [
+  '@mui',
+  '@uppy',
+  '@aws-amplify',
+  'imask',
+  'i18next',
+  'date-fns',
+  '@tanstack',
+  'isbn3',
+  'yup',
+  'axios',
+  'formik',
+];
+
+export default defineConfig({
+  plugins: [react(), eslint()],
+  server: {
+    open: true,
+    port: 3000,
+  },
+  preview: {
+    port: 3000,
+  },
+  build: {
+    outDir: 'build',
+    rollupOptions: {
+      onwarn: ({ message }) => {
+        if (message.startsWith('[plugin vite-plugin-eslint]')) {
+          throw new Error(message);
+        }
```

Comment:
overflødig?
```suggestion
        throw new Error(message);
        
```

Reply:
Kommer noen andre warnings som ikke er fra vår kode her også, så vi må nok ha senne sjekken. Legger i tillegg til logging av alle warnings, sånn at vi blir eksponert for dem